tests/unit/unit_utils.py

```diff
@@ -0,0 +1,13 @@
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+
+class StringIOContextManager(StringIO):
+    """
+    A StringIO that can be used as a context manager
```

Comment:
"A StringIO that can be used as a context manager" does not add any description to "StringIOContextManager"